web/src/components/tasks/business-formation/contacts/Addresses.tsx

```diff
@@ -179,80 +196,104 @@ export const Addresses = <T extends FormationMember | FormationIncorporator>(
+                                  onChange: (event) => {
+                                    return handleSignerCheckbox(event, index);
+                                  },
+                                  checked: it.signature,
+                                  fieldName: props.fieldName,
+                                  index,
+                                })}{" "}
+                              </div>
+                            </>
+                          ) : (
+                            <></>
+                          )}
+                        </span>
+                        <span>
+                          <span className="vl border-base-light padding-y-05 padding-right-2 margin-right-2">
+                            <IconButton
+                              aria-label="edit"
+                              onClick={(): void => {
+                                setEditIndex(index);
+                                setModalOpen(true);
+                              }}
+                              className="usa-button usa-button--unstyled width-auto"
+                            >
+                              <Icon className="usa-icon--size-3">edit</Icon>
+                            </IconButton>
+                          </span>
                           <IconButton
-                            aria-label="edit"
-                            onClick={(): void => {
-                              setEditIndex(index);
-                              setModalOpen(true);
-                            }}
+                            aria-label="delete"
+                            onClick={(): void => deleteAddress(index)}
                             className="usa-button usa-button--unstyled width-auto"
                           >
-                            <Icon className="usa-icon--size-3">edit</Icon>
+                            <Icon className="usa-icon--size-3">delete</Icon>
                           </IconButton>
                         </span>
-                        <IconButton
-                          aria-label="delete"
-                          onClick={(): void => deleteAddress(index)}
-                          className="usa-button usa-button--unstyled width-auto"
-                        >
-                          <Icon className="usa-icon--size-3">delete</Icon>
-                        </IconButton>
-                      </span>
-                    </div>
-                  </td>
-                </tr>
+                      </div>
+                    </td>
+                  </tr>
+                  {doesFieldHaveError(props.fieldName) && "signature" in it && !it.signature ? (
+                    <tr key={`error-${index}`}>
+                      <td className="flex-column text-error-dark text-bold">
+                        {Config.formation.fields.signers.errorBannerCheckbox}
+                      </td>
+                    </tr>
+                  ) : (
```

Comment:
I think this can be simplified to be 

{
{(doesFieldHaveError(props.fieldName) && "signature" in it && !it.signature) && (
                    <tr key={`error-${index}`}>
                      <td className="flex-column text-error-dark text-bold">
                        {Config.formation.fields.signers.errorBannerCheckbox}
                      </td>
                    </tr>
                  )
}